crates/evm/core/src/opts.rs

```diff
@@ -24,6 +24,9 @@ pub struct EvmOpts {
     /// pins the block number for the state fork
     pub fork_block_number: Option<u64>,
 
+    /// The number of retries
+    pub fork_retries: Option<u32>,
```

Comment:
we don't have a correspond cli arg for this like

https://github.com/foundry-rs/foundry/blob/c76d3723bad06aecaabb18afd158218e2a1958fb/crates/common/src/evm.rs#L58-L63

so this will also be None